### DISCUSSION TITLE
Will you support WatchOS in the future?

### DISCUSSION THREAD

**Discussion by bowencool** [tauri-apps/cargo-mobile2#62] [Ideas]
Will you support WatchOS in the future?

**Comment by FabianLars** [comment]
As far as i know watchos still only supports web content (via webkit) in the Mail and Messages app. If they make wkwebview generally available then i think this could be a really cool target for us to explore. I don't expect this to happen any time soon tho since web engines tend to drain these small batteries in no time 😅
